recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
     - windows_cmake_install_bindir.patch       # [win]
 
 build:
-  number: 0
-  # py36 since it uses vc14 like py35
-  skip: true  # [win and py36]
```

Comment:
Please add this `skip` and comment above back. They are meant to be here. Please see this [response]( https://github.com/conda-forge/gflags-feedstock/pull/7/files#r111460322 ) for details.